catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/forms/model/XmlModelBuilder.java

```diff
@@ -0,0 +1,467 @@
+/**
```

Comment:
Are the comments above the methods to remind you to write tests for those methods?

Reply:
I went back and refactored my test method names. Following that, I made a ticket for improving unit test coverage and replaced comments (like the above) with a link to the ticket. 

The `// ~` indicated it was a new validation method that didn't have a translation in the old naming scheme.

Reply:
https://codice.atlassian.net/browse/DDF-3832